version.go

```diff
@@ -19,27 +19,52 @@ import (
 	"fmt"
 	"strings"
 
+	hclversion "github.com/hashicorp/go-version"
 	tfversion "github.com/hashicorp/go-version"
+	"github.com/madlambda/spells/errutil"
+	"github.com/rs/zerolog/log"
 )
 
 //go:embed VERSION
 var version string
 
-var tfversionObj *tfversion.Version
-
-func init() {
```

Comment:
by removing this we allow `terramate.Version()` to potentially return invalid versions.

Reply:
which will fail if there is version checking being done, and if version check is not done not sure why we would need to panic/fail (if no version is defined on a project you are not interested in versions).

Reply:
will add some tests for when the check works, they should help us avoid releasing terramate with invalid version on Version() (right now it could happen), but still not sure we need to check/panic on init, if someone is not interested in version checking in a project it seems odd to do the check all the time anyway on init, we could add come build time safeguard to avoid ever releasing it with a wrong version, but it is not a runtime thing when the user is running terramate I think :thinking:

Reply:
I meant when used as a library. The `terramate.Version()` is exported and potentially can be invalid. We had early report of brew users directlying altering the VERSION file with a non-semver version.
But maybe adding a test in the e2etest is enough. Just pointing out it's a bit strange we allow it because the project depend on strictly semver version strings.